src/types/index.ts

```diff
@@ -44,9 +44,10 @@ export interface ArchaeologistProfile {
   exists: boolean;
   minimumDiggingFee: BigNumber;
   maximumRewrapInterval: BigNumber;
-  successes: string[];
-  cleanups: string[];
-  accusals: string[];
+  successes: BigNumber;
+  cleanups: BigNumber;
+  accusals: BigNumber;
+  failures: BigNumber;
```

Comment:
Wondering - are cleanups going to be treated as failures? I'm a bit behind on the current metrics structure.

Reply:
Yeah, cleanups are failures. The failure count here includes cleanups.